### DISCUSSION TITLE
Access $el when binding via x-spread

### DISCUSSION THREAD

**Discussion by haggen** [alpinejs/alpine#728] [4. General]
I found [an issue](https://github.com/alpinejs/alpine/issues/627) about accessing `$event` when binding handlers via `x-spread` but I couldn't anything about accessing `$el` in the same handler. Here's a contrived version of what I'm trying to achieve:

```js
function modal() {
  return {
    // state, open(), close(), etc...

    modal: {
      ['@modal:open.window']($event) {
        if ($event.detail === $el.id) { // $el here is undefined
          this.open();
        }
      }
    }
  }
}
```

**Comment by ryangjchandler** [comment]
@haggen Please try using `this.$el`.

**Reply by haggen** [reply]
Thank you, that worked.
